MANIFEST.in

```diff
@@ -0,0 +1 @@
+Include galaxy_ng/app/webserver_snippets/*
```

Comment:
`webserver_snippets` isn't getting included in the wheel. Might need to change `Include` -> `include`